### PR TITLE
Fabo/gaia 0.28.0 rc2

### LINKED ISSUE #1378: Steak is now called STAKE
find and replace `steak` with `stake`

### LINKED ISSUE #1646: Update wallet transactions
Description:

Was `/txs?tag=<tag>='<value>'` and now is `/txs?<tag>=<value>`

### LINKED ISSUE #1669: governance depositer was renamed to depositor
Description:

Part of the breaking changes from latest SDK release

### PR DESCRIPTION
**Pull request by faboweb**
closes #1378 
closes #1646 
closes #1669

SDK commit: `945803d586b83d65547cd16f4cd5994eac2957ea `

### CODE REVIEW

File: app/src/main/index.js [hunk 1]
```diff
@@ -271,10 +271,13 @@ async function startLCD(home, nodeURL) {
     ])
     logProcess(child, join(home, `lcd.log`))
 
-    child.stdout.once(`line`, async line => {
-      const certPath = /\(cert: "(.+?)"/.exec(line)[1]
-      resolve({ ca: fs.readFileSync(certPath, `utf8`), process: child })
-      lcdStarted = true
+    child.stdout.on(`line`, line => {
+      if (/\(cert: "(.+?)"/.test(line)) {
```
Comment (faboweb) [inline app/src/main/index.js]: Outputs lines before the certificate now

File: app/src/renderer/connectors/lcdClient.js [hunk 2]
```diff
@@ -199,19 +199,11 @@ const Client = (axios, localLcdURL, remoteLcdURL) => {
...
-          `GET`,
-          `/txs?tag=action='submit-proposal'&tag=proposer='${address}'`,
-          true
-        )(),
-        req(
-          `GET`,
-          `/txs?tag=action='deposit'&tag=depositer='${address}'`,
-          true
-        )()
+      let [depositorTxs, proposerTxs] = await Promise.all([
```
Comment (sabau) [inline app/src/renderer/connectors/lcdClient.js]: 👍

Comment (sabau) [reply app/src/renderer/connectors/lcdClient.js]: Minor: if you want we can write it simlarly as line 84 and store a variable less

File: app/src/renderer/components/wallet/PageWallet.vue [hunk 3]
```diff
@@ -115,7 +115,9 @@ export default {
         [`desc`, `asc`]
```
Comment (sabau) [inline app/src/renderer/components/wallet/PageWallet.vue]: Not sure how long this list can become (if less then 1000 ignore the following please :) )

Ordering is more expensive than filtering, so maybe we can order on top of the filtered list

Comment (faboweb) [reply app/src/renderer/components/wallet/PageWallet.vue]: On PageWallet we don't need to care about this for now. But very good insight!

**Comment by faboweb** [comment]
E2E tests still need to be fixed.

File: app/src/renderer/connectors/lcdClient.js (hunk 2 above)
Comment (faboweb) [reply app/src/renderer/connectors/lcdClient.js]: Done.

**Comment by fedekunze** [comment]
need to resolve conflicts

**Pull request merged**